src/start_location.h

```diff
@@ -27,6 +27,12 @@ struct start_location_placement_constraints {
     numeric_interval<int> allowed_z_levels{ -OVERMAP_DEPTH, OVERMAP_HEIGHT };
 };
 
+struct omt_types_parameters {
+    std::string omt;
+    ot_match_type omt_type;
+    std::vector<std::pair<std::string, std::string>> parameters;
```

Comment:
Is there a reason to use a vector of pairs instead of a map here?

Reply:
To begin with I were relying on iterators to match positions between two vectors for some reason but ye I don't think there's any need